habits/main.py

```diff
@@ -16,13 +16,13 @@ def on_startup():
     create_db_and_tables()
 
 
-def get_session():
-    with Session(engine) as session:
-        yield session
-
-
 @app.post("/users/", response_model=UserRead, status_code=status.HTTP_201_CREATED)
 def create_user(*, session: Session = Depends(get_session), user: UserCreate):
+    query = select(User).where(User.handle == user.handle)
```

Comment:
I wonder if we should move this to DB (`unique=True`) then catch the exception here?